### PR TITLE
Improved rigidity slider handling again

### LINKED ISSUE #3203: Rigidity slider can become disabled while there is still MP (multicellular?)
report came in that in multicellular the rigidity slider got disabled while there were 8 MP remaining. This might be multicellular specific unless my refactoring caused the microbe editor to also break like this.

Probably can be fixed by going through the actions in the editor and make sure it probably updates the rigidity state. Also when switching to the cell editor tab needs to make sure that the logic there will set things correctly for that tab.

### PR DESCRIPTION
**Pull request by 84634E1A607A**
**Related Issues**
Suppresses #1533
Closes #3452 
Fixes #3203

**Progress Checklist**

Note: before starting this checklist the PR should be marked as non-draft.

- [x] PR author has checked that this PR works as intended and doesn't
      break existing features:
      https://wiki.revolutionarygamesstudio.com/wiki/Testing_Checklist
      (this is important as to not waste the time of Thrive team
      members reviewing this PR)
- [x] Multicellular functionality needs testing
- [ ] Initial code review passed (this and further items should not be checked by the PR author)
- [ ] Functionality is confirmed working by another person (see above checklist link)
- [ ] Final code review is passed and code conforms to the 
      [styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md).

Before merging all CI jobs should finish on this PR without errors, if
there are automatically detected style issues they should be fixed by
the PR author. Merging must follow our
[styleguide](https://github.com/Revolutionary-Games/Thrive/blob/master/doc/style_guide.md#git).

### CODE REVIEW

**Comment by 84634E1A607A** [comment]
I think this's got bugs in multicellular. Wait until I figure this out.

**Comment by 84634E1A607A** [comment]
It works well actually...

File: src/microbe_stage/editor/CellEditorComponent.cs [hunk 1]
```diff
@@ -598,6 +598,7 @@ public override void OnEditorSpeciesSetup(Species species)
         // that just marks things dirty and we update our stats on the next _Process call)
         Membrane = Editor.EditedCellProperties!.MembraneType;
         Rigidity = Editor.EditedCellProperties.MembraneRigidity;
+        originalRigidity = (int)Math.Round(Rigidity * Constants.MEMBRANE_RIGIDITY_SLIDER_TO_VALUE_RATIO);
```
Comment (hhyyrylainen) [inline src/microbe_stage/editor/CellEditorComponent.cs:601]: Storing the original value like this goes against the dynamic MP, so having just the rigidity not follow the way everything else is handled in the editor is not the right design in my opinion.

**Comment by 84634E1A607A** [comment]
I tried but was defeated by the dynamic MP. I don't even know if a bug is because dynamic MP or because I wrote it. Closing this.

**Comment by hhyyrylainen** [comment]
I think I'll open sometime soon a discussion topic on the dev forum about the dynamic MP and if there is enough willingness to do a system overhaul on it.

**Pull request closed without merging**